core/store/src/config.rs

```diff
@@ -97,7 +97,7 @@ impl Default for StoreConfig {
             // we use it since then.
             block_size: bytesize::ByteSize::kib(16),
 
-            trie_cache_capacities: vec![(ShardUId { version: 1, shard_id: 3 }, 2_000_000)],
```

Comment:
Do we want to keep the functionality to override this per shard?

Reply:
I am worried about "per-shard" bit. From user's peerspective, "set `trie_cache_capacity: 90001`" is nice and simple. "set `trie_cache_capacities: [[{"verison": 1, "shard_id": 3}, 9000]]`" is way less user friendly.

Reply:
Also, did we actually shipped this? Should this be reveted on the next-release branch that? 

If we didn't completely ship the `trie_cache_capacities: [[{"verison": 1, "shard_id": 3}, 9000]]` config, and if it's not mandatory for fixing immediate problems, I would petition to remove the field in this form from the config completely, and spend some time shipping friendly and usable config. 

The specific thing I don't like is the config format -- it's just `derive(Serialize)`, which leads to quite user-unfriendly JSON.

Reply:
> Also, did we actually shipped this? Should this be reveted on the next-release branch that?

I’m going to cherry-pick this for 1.28 once this merges.